Jenkinsfile

```diff
@@ -2,22 +2,35 @@
 
 class GlobalVars { // "Groovy"
     public static boolean archiveArtifacts = false;
+    public static boolean isTestSuiteJob = false;
+    public static boolean COPY_INSTEAD_OF_BUILD_FIXME_REMOVE_THIS = true;
 }
 
 if (env.CHANGE_ID && !shouldBuildPullRequest()) {
     echo "Not building this pull request."
     return
 }
 
+echo("JOB_NAME='${env.JOB_NAME}', JOB_BASE_NAME='${env.JOB_BASE_NAME}'")
+def rateLimit = rateLimitBuilds([count: 1, durationName: 'day', userBoost: true])
```

Comment:
It appears that `rateLimitBuilds()` is broken with the current jenkins installation: `WARNING: Unknown parameter(s) found for class type 'jenkins.branch.RateLimitBranchProperty$JobPropertyImpl': count,durationName,userBoost`
This is also happening on the normal job so the ratelimit is not being applied :( 
Guess I'll have to open a Jenkins bug report.